core-java-8/src/main/java/com/baeldung/findanelement/FindElementInAList.java

```diff
@@ -0,0 +1,72 @@
+package com.baeldung.findanelement;
+
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Optional;
+import org.apache.commons.collections4.IterableUtils;
+import com.google.common.base.Predicate;
+import com.google.common.collect.Iterables;
+
+public class FindElementInAList<T> {
+
+    public T findUsingIndexOf(T element, List<T> listOfIntegers) {
+        int index = listOfIntegers.indexOf(element);
```

Comment:
Since it is templated, these aren't really a `listOfIntegers`. Let's change the name to something more generic.

Reply:
Sure, will update the name to list